package.json

```diff
@@ -19,8 +19,14 @@
     "@angular/platform-browser": "~8.0.1",
     "@angular/platform-browser-dynamic": "~8.0.1",
     "@angular/router": "~8.0.1",
+    "@ng-bootstrap/ng-bootstrap": "^5.0.0",
     "bootstrap": "^4.3.1",
+    "bulma": "^0.7.5",
+    "jquery": "^3.4.1",
+    "ng2-toasty": "^4.0.3",
+    "popper": "^1.0.1",
```

Comment:
likely do not need popper? were these installed because of warnings? if so delete them.